.travis.yml

```diff
@@ -93,4 +93,4 @@ script:
   - cmake ..
   - make -j
   - make check
-  - cd ../test && python -m unittest discover
+  - cd ../test/integration && python test_twemcache.py
```

Comment:
Because test discovery right now is broken (due to how I wrote runTest method), so I need to fix it before use that style again.